js/src/logger.ts

```diff
@@ -215,6 +215,18 @@ export interface Span extends Exportable {
    */
   permalink(): Promise<string>;
 
+  /**
+   * Format a link to the Braintrust application for viewing this span.
+   *
+   * Similar to permalink() but synchronous (no Promise).
```

Comment:
maybe mark permalink as deprecated?

Reply:
I will once it's fully deprecated. I have a few more steps to do to get there.

Reply:
Maybe mention the status or something in the comment then? Just want to make it easy for people to know which one to use between the two.